udf/mrn_udf_highlight_html.cpp

```diff
@@ -35,13 +35,19 @@ MRN_BEGIN_DECLS
 extern mrn::DatabaseManager *mrn_db_manager;
 extern mrn::ContextPool *mrn_context_pool;
 
+enum target_type {
+  TEXT = 0,
+  HTML = 1
```

Comment:
Could you append `MRN_TARGET_TYPE_` prefix? (We can remove them once we can use `enum class`.)

```suggestion
  MRN_TARGET_TYPE_TEXT = 0,
  MRN_TARGET_TYPE_HTML = 1
```

Reply:
Fixed.